src/constants/colors.js

```diff
@@ -7,3 +7,10 @@ export const textColor = "#272727";
 
 export const filterButtonColor = "#5C5C5C";
 export const filterButtontextColor = "#ADADAD";
+
+// EVENT DETAILS
+export const eventDetailsBgColor = "#fff";
+export const eventDetailsHeaderBgColor = "#F2F2F2";
+export const eventCategoryLabelBgColor = "#d8d8d8";
```

Comment:
Do we want to use uppercase characters for all colors?

Reply:
Yea that's cleaner, fixed.